### DISCUSSION TITLE
Mask placeholder color if "lazy: false"

### DISCUSSION THREAD

**Discussion by michael8113** [uNmAnNeR/imaskjs#814] [Q&A]
Hi!
How to set placeholder color if "lazy: false" and "mask: '+{0} (000) 000-00-00'"?
Need a different color for input text and mask placeholder =)

Thanks!

**Comment by uNmAnNeR** [comment]
you can use common css for it. mask is just about handling input and not any styling

**Reply by R0LLeX** [reply]
but how to set different colors for the mask and for the text entered in the input?

**Reply by uNmAnNeR** [reply]
no way for now. you can try to use content editable but it's just my theory.

**Reply by schen-fn** [reply]
Is this a feature that you can add in future releases?

**Reply by uNmAnNeR** [reply]
@schen-fn i don't think it's possible to have multiple styles for text in the input

**Reply by amytangcodes** [reply]
Is there a way to maybe set `lazy: false`/ imask placeholder to be passed through actual attribute `placeholder:` this way you can `input::placeholder` to define unique styling?
